Updating CallRecords API v1.0 documentation - New fields added (#21535)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/callrecords-mediastream.md

|averageAudioDegradation|Double|Average Network Mean Opinion Score degradation for stream. Represents how much the network loss and jitter has impacted the quality of received audio.|
|averageAudioNetworkJitter|Duration|Average jitter for the stream computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
|averageBandwidthEstimate|Int64|Average estimated bandwidth available between two endpoints in bits per second.|

> |averageFreezeDuration|Duration|Average of the received freeze duration related to the video stream.|

Reviewer: ```suggestion
|averageFreezeDuration|Duration|Average duration of freezing in the video stream.|
```

|maxRoundTripTime|Duration|Maximum network propagation round-trip time computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
|packetUtilization|Int64|Packet count for the stream.|
|postForwardErrorCorrectionPacketLossRate|Double|Packet loss rate after FEC has been applied aggregated across all video streams and codecs.|

> |rmsFreezeDuration|Duration|Root mean square of the average video stream freezing duration.|

Reviewer: ```suggestion
|rmsFreezeDuration|Duration|Average duration of the received freezing time in the video stream represented in root mean square.|
```

> |rmsFreezeDuration|Duration|Root mean square of the received freeze duration related to the video stream.|

Reviewer: ```suggestion
|rmsFreezeDuration|Duration|Root mean square of the average video stream freezing duration.|
```
